autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/sync/SyncDateProvider.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.sync
+
+import org.threeten.bp.OffsetDateTime
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.format.DateTimeFormatter
+
+class SyncDateProvider {
+    companion object {
```

Comment:
nit: it might make things easier to test if this was a interface we injected in DI, and could replace it with a double during testing. we do that elsewhere, e.g.

```
interface TimeProvider {
    fun currentTimeMillis(): Long
}

@ContributesBinding(AppScope::class)
class SystemCurrentTimeProvider @Inject constructor() : TimeProvider {
    override fun currentTimeMillis(): Long = System.currentTimeMillis()
}
```

 feel free to ignore if testing what happens at different times isn't a problem

Reply:
Just introduced this wrapper for clarity when reading the code, I'm good not injecting this for now.